tempesta_fw/http_msg.c

```diff
@@ -967,3 +973,57 @@ __tfw_http_msg_alloc(int type, bool full)
 	return hm;
 }
 
+TfwHttpMsg *
+tfw_http_msg_alloc_stream_part(TfwHttpMsg *hm_head)
+{
+	TfwHttpStream *stream;
```

Comment:
I supposed that if we receive a part of streamed HTTP message, then there is current `TfwHttpMsg` which we can attach the new data, use it for the parser and so on. Why do we need `TfwHttpStream` and even allocate the new descriptor?